department-of-veterans-affairs/caseflow
# Craig/appeals 8287

Resolves [APPEALS-8287](https://vajira.max.gov/browse/APPEALS-8287)

### Description
- Modified db/seeds/users.rb to create five more JudgeTeams using data that already exists in the VACOLS and Caseflow DB seed data
- Replaced randomized values with fixed ones in db/seeds/priority_distributions.rb, and removed code that modified existing JudgeTeams
- Added the db/seeds/priority_distributions.rb seed to the db/seeds.rb file so that it is run automatically when db:seed is run

### Acceptance Criteria
- [ ] PushPriorityAppealsToJudgesJob.perform_now runs without errors

### Testing Plan
1. Reseed the Caseflow and VACOLS DB (`make reset`; this can take 30 minutes)
2. Open the rails console (`make c`)
3. Run this command to set the current user for the PaperTrail gem: `RequestStore[:current_user] = User.system_user`
4. Run `PushPriorityAppealsToJudgesJob.perform_now`
5. Verify that the above job runs with no errors and returns 'nil' on completion
6. Verify that priority case distributions occurred in the database by checking the 'distributions' table and verifying that there are 12 new distributions with statistics that include a "priority_count" hash that is decreasing with each successive distribution.
